src/main/groovy/io/nextflow/gradle/registry/RegistryClient.groovy

```diff
@@ -25,7 +25,7 @@ class RegistryClient {
     }
 
     def publish(String id, String version, File file) {
-        def req = new HttpPost(url.resolve("publish"))
+        def req = new HttpPost(url.resolve("v1/plugins/publish"))
```

Comment:
The slash in the path concatenation is a classic bug (not saying this is buggy). it would better to have a little helper handling both presence and absence of slash and validating with a unit test

Reply:
Fair